website/docs/zephyr-training/05_golioth/west_manifest.md

```diff
@@ -43,7 +43,30 @@ manifest file responsible for installing its own copy of the Zephyr tree and all
 dependencies. This manifest file is committed to the project repo, so that the
 build state is known (and replicable) for every commit in that repo.
 
-This is the approach that we used for [the Zephyr Training code
+<details>
+    <summary>Explain why I should be using standalone project manifests</summary>
+
+Compare a Standalone Project to building your project in the samples directory
```

Comment:
Just in case you wanted to use the Zephyr terminology for these, the Zephyr docs call these "repository apps" vs. "workspace apps" (they also define a "freestanding app"): https://docs.zephyrproject.org/latest/develop/application/index.html#application-types